examples/statistics/histogram_histtypes.py

```diff
@@ -17,19 +17,31 @@
 
 np.random.seed(19680801)
 
-mu = 200
-sigma = 25
-x = np.random.normal(mu, sigma, size=100)
+mu_x = 200
+sigma_x = 25
+x = np.random.normal(mu_x, sigma_x, size=100)
 
-fig, (ax0, ax1) = plt.subplots(ncols=2, figsize=(8, 4))
+mu_w = 200
+sigma_w = 10
+w = np.random.normal(mu_w, sigma_w, size=100)
 
-ax0.hist(x, 20, density=True, histtype='stepfilled', facecolor='g', alpha=0.75)
-ax0.set_title('stepfilled')
+fig, axs = plt.subplots(nrows=2, ncols=2)
+
+axs[0, 0].hist(x, 20, density=True, histtype='stepfilled', facecolor='g', alpha=0.75)
```

Comment:
This line is too long, hence the test error

    ./examples/statistics/histogram_histtypes.py:30:80: E501 line too long (85 > 79 characters)

Reply:
Oops, I have fixed the line. The CircleCI build was failing once I did this due to a missing import fixed in 4677fd9f42d3cb0b38f09da40a38435650f134c4